sass/themes/okc/_theme.scss

```diff
@@ -15,18 +16,19 @@ $viewpoint-marker-color: $primary-color;
 $current-location-color: $primary-color;
 
 $font-family: Arial, sans-serif;
+$font-family-narrow: $font-family;
```

Comment:
The name `font-family-narrow` implies that the variable always causes text to be narrow, but it really just contains whatever `$font-family` contains. Is that intended?

Reply:
OKC currently does not define a narrow font style. However, DT at various places uses narrow font families. For now, the normal font-family is intended.